### ISSUE TITLE
Exercise 1 -- Domain of n

### ISSUE THREAD

**Issue by alyfarahat** [uwhpsc-2016/homework1#3]
It is mentioned that n is in N. I suppose that for the Collatz conjecture to work, n should be a positive integer. Isn't that true? My understanding is that N is the set of non-negative integers.

**Comment by cswiercz** [comment]
Yes! Good catch. The domain of the function C should be the non-zero natural numbers, N_+, not the natural numbers.

**Comment by jhyearsley** [comment]
Depends which country you are in! For instance in France, the set of natural numbers includes zero whereas in the United States the standing convention is that the natural numbers start at one. Silly!

**Comment by quantheory** [comment]
I'm not sure that it's consistent even within the U.S. When I was little (in middle school?), I think I was told that that natural numbers start at 1, and the "whole" numbers start at 0. But after high school, I don't think that I really heard anyone talk about the "whole" numbers at all, and when people talked about natural numbers they usually included 0. It's just ambiguous!

**Comment by jhyearsley** [comment]
Agreed!

**Comment by ckchmod** [comment]
This is an interesting conversation. A professor once told me, the definition of N is so befuddled now, that on the front of most textbooks/research papers, people often have to clarify and state what THEY mean by natural numbers. Kinda like in graph theory, a lot of terminologies overlap now, so they redefine everything before introducing new concepts.

**Comment by afarahatoutset** [comment]
In programming jargon, they call that an "off-by-one" bug; they did not specify a sign for one, though.
